README.md

```diff
@@ -13,6 +13,7 @@ DO Agent currently supports:
 - Debian ([oldest supported](https://wiki.debian.org/LTS) LTS release and later)
 - Fedora 27+
 - CentOS 6+
+- CloudLinux 6+
```

Comment:
Remove this from this section and add another section below called "not officially supported" or something like that.

Reply:
@truls1502 we can merge this PR in, if you address this comment and the other outstanding one. Let us know if you are still interested in getting this merged.

Reply:
Done.